gef-ui/server/controllers/supporting-information/manual-inclusion-questionnaire/index.js

```diff
@@ -90,6 +90,7 @@ const postUploadManualInclusion = async (req, res) => {
       validFiles,
       FIELD_NAME,
       application,
```

Comment:
Havent updated implemtation to pass id rather than deal?

Reply:
Good catch, thanks. Sorry juggling this and errors in numgen